docs/tutorials/scala/README.md

```diff
@@ -1,6 +1,34 @@
 # MXNet-Scala Tutorials
 
-* [Setup Scala with MXNet and Create a MXNet-Scala Project with IntelliJ](mxnet_scala_on_intellij.md)
+## Installation & Setup
+
+Using MXNet-Scala is easiest with Maven. Refer to the following tutorial to see how you can use Maven with IntelliJ.
+
+* [Setup Scala with MXNet and Create a MXNet-Scala Project with IntelliJ (macOS)](mxnet_scala_on_intellij.md)
+
+You can also build the Scala package yourself.
+
+### Build the MXNet Shared Library and Scala Package
+
+This depends on your operating system. Instructions for macOS, Ubuntu, and Windows are provided:
+
+| OS | Step 1 | Step 2 |
+|---|---|---|
+|macOS | [Shared Library for macOS](http://mxnet.incubator.apache.org/install/osx_setup.html#build-the-shared-library) | [Scala Package for macOS](http://mxnet.incubator.apache.org/install/osx_setup.html#install-the-mxnet-package-for-scala) |
```

Comment:
Not any more. You don't need build from source

Reply:
Shall we keep the build from source as an alternative, if there are developers trying to use the most recent package?